tests/test-nima-jsonb/src/main/java/org/example/TestController.java

```diff
@@ -53,14 +55,30 @@ String enumQueryImplied(String s, @QueryParam ServerType type) {
   }
 
   @InstrumentServerContext
-  @Get(value =  "/inputStream")
+  @Get(value = "/inputStream")
   InputStream stream(InputStream stream) throws Exception {
     return stream;
   }
 
-
   @Post("/strBody")
   String strBody(@BodyString String body) {
     return body;
   }
+
```

Comment:
Lets merge this. I'm just thinking that in general exception handling is "global" and currently we attach it to controllers.

I'm thinking that ideally exception handling is added to a "root controller" or perhaps ... we allow target=class for `@ExceptionHandler` as a kind of "Exception Handler controller" that we expect all exception handlers to be added to (and not add them to controllers per se).

Reply:
I could add a `@ControllerAdvice` annotation if you want

Reply:
Hmm, let's not add another annotation just let.

Reply:
> add a @ControllerAdvice annotation if you want

FYI:   Just to say we can just have a `@Controller` that _ONLY_ has `@ExceptionHandler` methods on it and no `@Path` so at this stage I think I am happy with that and I think that makes decent sense.